frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1340,7 +1360,9 @@ function ReaderFooter:addToMainMenu(menu_items)
                 },
             },
             {
-                text = _("Item separator"),
+                text_func = function()
+                    return T(_("Item separator: '%1'"), self:get_separator_symbol())
```

Comment:
I'm not sure the symbol needs to be shown inside quotes. `Item separator: .` would look just fine.

Reply:
OK, then we should show `none` if none is seleceted.